src/Tests/Tests/Test_FastConv.cpp

```diff
@@ -23,19 +25,158 @@ namespace fastconv_test {
     protected:
         void SetUp() override
         {
+            m_pCFastConv = new CFastConv();
```

Comment:
This "new" doesn't have a corresponding "delete", and could lead to memory leaks

Reply:
I'm not sure why, but if I delete m_pCFastConv in the TearDown function, the tests crash...